src/index.js

```diff
@@ -1,7 +1,64 @@
-require('file-loader?name=[name].[ext]!./index.html');
+require('file-loader?name=[name].[ext]!./front_end/index.html');
+import './style.css';
 
 const heading = document.getElementById("heading");
+var data = "";
+var patientId = "";
 
-if (heading != null) {
-    heading.textContent = "hi";
+patientId = localStorage.getItem("patientID");
+
+function httpGet(theUrl)
```

Comment:
deze heb ik vaker gezien --> in utils dus! (onder services)

Reply:
nee, dan werkt het niet

Reply:
hoezo werkt het niet? lijkt me dat je dat ding gwn kunt verplaatsen?

Reply:
nee heel veel gedoe mee gehad en heel lang naar gezocht maar het werkt niet dus heb ik het zo gedaan